### PR TITLE
Bump PF version and get rid of some custom CSS styles

### LINKED ISSUE #870: [Deps]: Bump version of PatternFly to 2022.16 release
### Feature description

"@patternfly/patternfly": "4.221.2" -> "4.222.4",
"@patternfly/react-catalog-view-extension": "4.93.0" -> "4.93.15",
"@patternfly/react-core": "4.264.0" -> "4.267.6",
"@patternfly/react-icons": "4.93.0" -> "4.93.3",
"@patternfly/react-styles": "4.92.0" -> "4.92.3",
"@patternfly/react-table": "4.111.45" -> "4.112.6",
"@patternfly/react-virtualized-extension": "4.88.67" -> "4.88.82",

### Describe alternatives you've considered

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by DaoDaoNoCode**
## Description

Closes #870 

## How Has This Been Tested?

Navigate around pages to see if anything changes.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by lucferbux** [comment]
/lgtm
Tested and i saw now issues, we need to check the smoke tests in case we are removing some class they are using (and then follow on the conversation about testing)

**Review by lucferbux — APPROVED** [review]

File: frontend/src/pages/exploreApplication/ExploreApplications.tsx [hunk 1]
```diff
@@ -58,7 +52,7 @@ const ExploreApplicationsInner: React.FC<ExploreApplicationsInnerProps> = React.
             />
           }
         >
-          <DrawerContentBody className={bodyClasses}>
+          <DrawerContentBody style={{ display: 'flex', flexDirection: 'column' }}>
```
Comment (andrewballantyne) [inline frontend/src/pages/exploreApplication/ExploreApplications.tsx:55]: A little concerning we are doing this -- can we not get a proper layout in place to handle flex? Why are we directly mutating a PF component to do this?

File: frontend/src/pages/projects/notebook/NotebookImagePackageDetails.tsx [hunk 2]
```diff
@@ -28,9 +28,9 @@ const NotebookImagePackageDetails: React.FC<NotebookPackageDetailsProps> = ({
         <DescriptionListTerm>{title || 'Packages'}</DescriptionListTerm>
         <DescriptionListDescription>
           {dependencies.map(getNameVersionString).map((pkg) => (
-            <p key={pkg}>
-              <Truncate content={pkg} />
-            </p>
+            <div key={pkg}>
```
Comment (andrewballantyne) [inline frontend/src/pages/projects/notebook/NotebookImagePackageDetails.tsx:31]: Why the change from a paragraph block?

**Review by andrewballantyne — CHANGES_REQUESTED** [review]
You're doing some layout changes to bring css padding/margin overrides into React. I don't disagree with this choice, but I want to make sure we don't do this going forward unless a specific exception is made. I'd prefer to lean into PF styles fully vs us overriding them.

File: frontend/src/pages/exploreApplication/ExploreApplications.tsx (hunk 1 above)
Comment (DaoDaoNoCode) [reply frontend/src/pages/exploreApplication/ExploreApplications.tsx:55]: I tried to apply the `Flex` layout to the DrawerContentBody, but it doesn't work out well, it will cause the page scrollable even if there are only a few cards on the page. I am trying to find out the reason, but I guess it's because the `section` element is not the child of DrawerContentBody anymore, which causes it inherits some different height that causes the expansion of the page height.

File: frontend/src/pages/projects/notebook/NotebookImagePackageDetails.tsx (hunk 2 above)
Comment (DaoDaoNoCode) [reply frontend/src/pages/projects/notebook/NotebookImagePackageDetails.tsx:31]: hmm, good catch, I might do some tests that time and forgot to change it back. Will revert this.

File: frontend/src/pages/exploreApplication/ExploreApplications.tsx (hunk 1 above)
Comment (andrewballantyne) [reply frontend/src/pages/exploreApplication/ExploreApplications.tsx:55]: 🤔 Hmmm -- well try to do away with the flex. I could help take a look if you're out of ideas. We shouldn't really re-associate PF components with a different display type.

Comment (DaoDaoNoCode) [reply frontend/src/pages/exploreApplication/ExploreApplications.tsx:55]: So if I go without any custom inline styles, it will be like: 
Screenshot 2023-01-09 at 3 58 13 PM
The content page section cannot fill the page up because the DrawerContentBody is not flex layout.

If I go with the current change, which is the custom inline styles, it will be like:
Screenshot 2023-01-09 at 3 59 51 PM
This will fill the page height up and not scrollable, which is good.

If I use the PF Flex layout instead, the content becomes scrollable, which I cannot find out the reason:
Screenshot 2023-01-09 at 4 02 21 PM

I will appreciate that if you could help me take a look at it! @andrewballantyne

Comment (andrewballantyne) [reply frontend/src/pages/exploreApplication/ExploreApplications.tsx:55]: Thanks for the explanation @DaoDaoNoCode I'll take a look tomorrow.

Comment (andrewballantyne) [reply frontend/src/pages/exploreApplication/ExploreApplications.tsx:55]: Okay, missed taking a look at this yesterday -- I think the issue boils down to the lack of `Page` when using `PageSection`s and you're reacting to that by mimicking the part of `Page` the `PageSection` needs -- which is Flex.

We should talk about solutions tomorrow.

File: frontend/src/pages/exploreApplication/ExploreApplications.tsx [hunk 3]
```diff
@@ -20,7 +19,7 @@ import { fireTrackingEvent } from '../../utilities/segmentIOUtils';
 import { ODH_PRODUCT_NAME } from '../../utilities/const';
 import { useAppContext } from '../../app/AppContext';
 
-import './ExploreApplications.scss';
+import 'DrawerContentBody.scss';
```
Comment (LaVLaS) [inline frontend/src/pages/exploreApplication/ExploreApplications.tsx:22]: npm build is failing due to this requiring explicitly referencing this module exists in the same directory

```suggestion
import './DrawerContentBody.scss';
```

**Review by andrewballantyne — APPROVED** [review]
